docs/index.md

```diff
@@ -0,0 +1,20 @@
+# Jenkins-k8s Operator
+
+A Juju charm deploying and managing Jenkins on Kubernetes. [Jenkins](https://www.jenkins.io/) is an open source automation server. Jenkins provides hundreds of plugins to support building, deploying and automating any project.
```

Comment:
Do you think this should mention that it can be integrated with both k8s and machine agents? Probably something on user's mind

Reply:
Added
>It can be integrated with both k8s and machine (VM) agents for use.
Thank you!